packages/stack-shared/src/interface/crud/users.ts

```diff
@@ -35,41 +56,6 @@ export const usersCrudServerReadSchema = fieldSchema.yupObject({
    * @deprecated
    */
   requires_totp_mfa: fieldSchema.yupBoolean().required().meta({ openapiField: { hidden: true, description: 'Whether the user is required to use TOTP MFA to sign in', exampleValue: false } }),
-  /**
-   * @deprecated
-   */
-  oauth_providers: fieldSchema.yupArray(fieldSchema.yupObject({
-    id: fieldSchema.yupString().required(),
-    account_id: fieldSchema.yupString().required(),
-    email: fieldSchema.yupString().nullable(),
-  }).required()).required().meta({ openapiField: { hidden: true, description: 'A list of OAuth providers connected to this account', exampleValue: [{ id: 'google', account_id: '12345', email: 'john.doe@gmail.com' }] } }),
-  auth_methods: fieldSchema.yupArray(fieldSchema.yupUnion(
```

Comment:
was there no place where we used auth_methods and connected_accounts in the frontend?

Reply:
I don't think they are used, I can double check